hybrid-cloud-patterns/qiot
# common automatic update

- Validation Schema for common/clustergroup
- - Fixed issues with schema validation in values.schema.json - Modified clustergroup/values.yaml and examples/values-example.yaml to have same sections and values. - Updated tests for  tests/clustergroup-naked.expected.yaml and tests/clustergroup-normal.expected.yaml
- Fixed issues with kubeconform in clustergroup/values.yaml.  Reran make test to generate proper files
- - Updated schema to include all elements in subscriptions - Updated tests associated with clustergroup - Updated values-example.yaml
- applications and subscriptions can be obj and array
- restore values.yaml file to a saner commented default
- Ran values.schema.json through jq to fix indentation issues
- Fix up tests
- Move to draft 0.7 of the schema spec
- Add operatorgroupExcludes which is in examples/industrial-edge-factory.yaml
- namespace is not mandatory in an application
- Fix up some more tests
- add clusterSelector in managedClusterGroups
- Allow clusterRoleYaml to be an array and timeout to be a string
- Add image, tags, extravars to imperative job object
- Add descriptions for SecretStore properties
- move GlobalGit and GitSecret after Main
- Forgot ,
- Fix up the main section a bit
- Make subscriptions not mandatory
- Add description to main.git fields
- Drop useCSV from global
- Add description for main.gitops.channel
- Add some descriptions
- Mark useCSV as deprecated
- - Added descriptions for namespaces, subscriptions, managedClusterGroup, applications, and projects
- - Fixed issue with description definition
- - Removed insecure boolean from schema file and from values-example.yaml
- Re-ran make tests
- Drop any remains of output_file
- Updated python-version for json schema
- Update python version for ansible-unittest
- Test fixes
